.github/workflows/run-functional-tests.yml

```diff
@@ -29,6 +31,12 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Check runFunctionalTests label
+        if: contains(github.event.pull_request.labels.*.name, 'runFunctionalTests')
+        run: |
+          echo "Add the runFunctionalTests label to the PR to trigger the Functional Tests workflow."
+          exit 1
```

Comment:
What is this supposed to do?

Reply:
ah, good catch
changed to "!contains...'
so if no appropriate label - fail step and job